tests/CMakeLists.txt

```
@@ -97,20 +97,21 @@ ENDFUNCTION()
 
 # Analyze the .prm to decide if this test should be run or not. We are
 # checking for the following strings in the .prm:
-# 1. "WORLD BUILDER" - only run with World Builder.
-# 2. "QUICK_TEST" - enable the test even if RUN_ALL_TESTS is false
+# 1. "Enable if: ASPECT_WITH_WORLD_BUILDER" - only run with World Builder.
+# 2. "Enable if: QUICK_TEST" - enable the test even if RUN_ALL_TESTS is false
 FUNCTION(SHOULD_ENABLE_TEST _filename)
 
   FILE(STRINGS ${_filename} _input_lines
-  REGEX "WORLD BUILDER")
+       REGEX "Enable if: +ASPECT_WITH_WORLD_BUILDER")
```

```suggestion
       REGEX "Enable if: ASPECT_WITH_WORLD_BUILDER")
```
or add the + for quick tests as well to be consistent.